src/XML/src/FMI3/fmi3_xml_variable.c

```diff
@@ -949,6 +886,43 @@ fmi3_uint64_t fmi3_xml_get_clock_variable_shift_counter(fmi3_xml_clock_variable_
     return props->shiftCounter;
 }
 
+// -----------------------------------------------------------------------------
+// Alias variable
+// -----------------------------------------------------------------------------
+
+fmi3_xml_alias_variables_t* fmi3_xml_get_variable_aliases(fmi3_xml_variable_t* v) {
+    if (v->aliases) {
+        return (void*)v->aliases;  // cast to opaque type
+    }
+    return NULL;
+}
+
+size_t fmi3_xml_get_alias_variables_number(fmi3_xml_alias_variables_t* aliases) {
+    if (aliases) {
+        return jm_vector_get_size(jm_voidp)(&aliases->vec);
+    }
+    return 0;
+}
+
+fmi3_xml_alias_variable_t* fmi3_xml_get_alias(fmi3_xml_alias_variables_t* aliases, size_t index) {
+    if (!aliases || index > jm_vector_get_size(jm_voidp)(&aliases->vec)) {
```

Comment:
There isn't, but thinking again I don't think we should check for index here, but rather let it crash. That's the way that lists usually handle it (probably due to performance).

I mean, imagine we just returned an array instead, then it would segfault if it's NULL or if index is out of bounds, because it's a programming error

Reply:
I removed the index check

Reply:
I suppose we should pick up this discussion at another point, since there are a bunch of these types of functions and various of them do contain the bounds checks. The behavior should be more consistent then.

Reply:
I agree, added cleanup (design) point about it.